pkg/staticnetworkconfig/backward_compatibility.go

```diff
@@ -0,0 +1,5 @@
+package staticnetworkconfig
+
+const MinimalVersionForNmstatectl = "4.13"
+
+const MaximalVersionForNmstatectl = "4.17"
```

Comment:
It was decided to use a temporary version range from 4.13 to 4.17 until we fetch the binary from rootfs. Later, we will adjust this to include version 4.13 and above.

Reply:
@AlonaKaplan correct me if I'm not accurate, but basically we decided on using a version range as a precaution in case we don't continue with this project in future versions. We're working in phases, and fetching the binary will be part of the next phase. For now, we're setting a temporary range, and later we can adjust it to include version 4.13 and above.

@AlonaKaplan  what do you think?

Reply:
As @carbonin mentioned, I don't think it worth maintaining a max version just to remove it later on. I.e. as we install the x86 version of nmstate when building the image-service, the arch check should be enough. If 4.18 is failing with the new flow for some weird reason, we should catch it early on anyway with the pre-release builds. Which sounds better than 'hiding' it with this condition.

Reply:
Sounds reasonable to remove the max version.